test/unit/utils/sentry.js

```diff
@@ -1,8 +1,31 @@
 /* eslint-env mocha */
 
-require('should')
+const should = require('should')
+const fs = require('fs')
 
 const sentry = require('../../../core/utils/sentry')
+const { FetchError } = require('electron-fetch')
+
+class CozyClientFetchError extends Error {
+  constructor(res, reason) {
+    super()
+    if (Error.captureStackTrace) {
+      Error.captureStackTrace(this, this.constructor)
+    }
+    // XXX We have to hardcode this because babel doesn't play nice when extending Error
```

Comment:
Babel? I thought babel was no longer used on cozy-desktop

Reply:
Ah, I just copy/pasted this class from `cozy-client-js` as it's not exported. I'll remove this line and add a comment to explain where the class is coming from.